src/pmdas/hdb/Install

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+#
+# Copyright (c) 2021 Red Hat.
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# See <http://www.gnu.org/licenses/> for copy of the GNU General Public License.
+#
+# Install the hdb PMDA
+#
+
+# Check the python hdbcli package is installed
+if ! pip3 show hdbcli > /dev/null 2>&1; then
```

Comment:
Yep, close - best to do exactly what the PMDA does though - run PCP_PYTHON_PROG with each of the non-default modules.  The activemq PMDA Install script is an example of doing this (except its with perl), and the configure.ac scripting has several python examples, e.g.

```
        dnl rpm build for mongodb PMDA needs python pymongo module
        if $pcp_python_prog -c "import pymongo" >/dev/null 2>&1
```